front-end-poc/yarn.lock

```diff
@@ -1162,6 +1162,22 @@
     "@types/istanbul-reports" "^1.1.1"
     "@types/yargs" "^13.0.0"
 
+"@manstis/pmml-linear-model-viewer@file:../front-end-poc-pmml-linear-model-viewer":
```

Comment:
IDK if we want to commit `yarn.lock`?

Reply:
I don't think it breaks anything, I would commit it. We will take care of that obviously when the module will move.